### PR TITLE
2020 12 01 tarkansevilmis planner feature update

### PR DESCRIPTION
**Pull request by TarkanSevilmis**
Updates for the Planner API.
Adding containers, deprecating owner on plan.
Adding task creation source info.

### CODE REVIEW

File: api-reference/beta/resources/plannerplan.md [hunk 1]
```diff
@@ -29,9 +29,10 @@ The **plannerPlan** resource represents a plan in Microsoft 365. A plan can be o
 ## Properties
 | Property	   | Type	|Description|
 |:---------------|:--------|:----------|
+|container|[plannerPlanContainer](../resources/plannerplancontainer.md)|Required. Identifies the container of the plan. After it is set, this property can’t be updated.|
 |createdDateTime|DateTimeOffset|Read-only. Date and time at which the plan is created. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`|
 |id|String| Read-only. ID of the plan. It is 28 characters long and case-sensitive. [Format validation](tasks-identifiers-disclaimer.md) is done on the service.|
-|owner|String|ID of the [Group](group.md) that owns the plan. A valid group must exist before this field can be set. After it is set, this property can’t be updated.|
+|owner|String|Deprecated. Please use *container* property instead. ID of the [Group](group.md) that owns the plan. A valid group must exist before this field can be set. After it is set, this property can’t be updated.|
```
Comment (Lauragra) [inline api-reference/beta/resources/plannerplan.md]: Please make sure that you're following the guidance here for deprecated properties: https://msazure.visualstudio.com/One/_wiki/wikis/Microsoft%20Graph%20Partners/73842/Deprecate-docs?anchor=deprecate-content-for-a-property-or-relationship

File: changelog/Microsoft.Office.Tasks.json [hunk 2]
```diff
@@ -197,6 +197,72 @@
       "CreatedDateTime": "2017-04-01T00:00:00",
       "WorkloadArea": "Tasks and plans",
       "SubArea": ""
+    },
+    {
```
Comment (Lauragra) [inline changelog/Microsoft.Office.Tasks.json]: In addition to the JSON file, we still need to update changelog.md as well, while the new filterable changelog experience is still in preview.

**Review by Lauragra — CHANGES_REQUESTED** [review]
I left a couple of comments - pls. take a look.

File: changelog/Microsoft.Office.Tasks.json [hunk 3]
```diff
@@ -1,5 +1,71 @@
...
+          "ChangedApiName": "plannerContainerType",
+          "ChangeType": "Addition",
+          "Description": "Added the **plannerContainerType** enumeration type.",
+          "Target": "plannerContainerType"
+        },
+        {
+          "Id": "a440f891-340e-11eb-9836-6aeb5ee7dda6",
+          "ApiChange": "Property",
+          "ChangedApiName": "owner",
+          "ChangeType": "Change",
```
Comment (paulodero) [inline changelog/Microsoft.Office.Tasks.json:57]: ChangeType for deprecated elements is `Deprecation`

File: changelog/Microsoft.Office.Tasks.json [hunk 4]
```diff
@@ -1,5 +1,71 @@
...
+          "Description": "Deprecated the **owner** property on the [plannerPlan](https://docs.microsoft.com/en-us/graph/api/resources/plannerPlan?view=graph-rest-beta) resource.",
```
Comment (paulodero) [inline changelog/Microsoft.Office.Tasks.json:58]: This is not the case. The property has not been deprecated in the schema change submitted.

**Review by paulodero — CHANGES_REQUESTED** [review]

File: changelog/Microsoft.Office.Tasks.json (hunk 4 above)
Comment (TarkanSevilmis) [reply changelog/Microsoft.Office.Tasks.json:58]: Added the schema annotation

**Review by Lauragra — APPROVED** [review]

**Pull request merged**